packages/xsnap/test/test-boot-lockdown.js

```diff
@@ -49,12 +51,12 @@ async function bootWorker(name, script, savePrinted = false) {
  * @param {boolean} [savePrinted]
  */
 async function bootSESWorker(name, savePrinted = false) {
-  const bootScript = await ld.asset('../dist/bundle-ses-boot.umd.js');
+  const bootScript = await ld.asset(bundlePaths.sesBoot);
```

Comment:
anybody remember why we used UMD here? It seems weird to not use the standard bundle here.

Reply:
best I can think of is that you don't have to wrap the UMD form in the IIEF magic, you just `eval` it. I'll see if I can make it behave like the others, and maybe then get rid of the UMD form, unless something is using it that my grep failed to find.

Reply:
I've changed all the callers to do the IIFE wrapping, and removed the UMD bundles